lib/matplotlib/axes/_base.py

```diff
@@ -3397,6 +3397,13 @@ def get_xticklabels(self, minor=False, which=None):
         -------
         ret : list
            List of `~matplotlib.text.Text` instances.
+        
+        Notes
+        -----
+        Please note that the tick label list is not populated until a ``draw``
```

Comment:
Maybe more precisely:`.. the tick label strings are not not...`

Reply:
OK